### PR TITLE
Add Tater to SSB

### PR DESCRIPTION
**Pull request by RagiingAlways**
Added Tater

### CODE REVIEW

**Comment by RagiingAlways** [comment]
There was an error - Fixed

**Comment by HoeenCoder** [comment]
Will review for errors in the morning

File: mods/sgssb/moves.js [hunk 1]
```diff
@@ -445,5 +445,20 @@ exports.BattleMovedex = {
...
     },
-
+    //CelestialTater
+     shellbreak: {
+     category: "Status",
+     id: "shellbreak",
+     isNonstandard: true,
+     name: "Shell Break",
+     boosts: {spa: 2, atk: 2, spe: 2, def: -1, spd: -1, accuracy: 1},
+     pp: 8,
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js]: PP should be the base PP (5). PS will "use a PP Max" on the move moving it to 8 PP.

File: mods/sgssb/scripts.js [hunk 2]
```diff
@@ -161,6 +161,12 @@ exports.BattleScripts = {
                 signatureMove: 'Ancient Orb',
                 evs: {atk: 252, spe: 252, hp: 4}, nature: 'Adamant',
             },
+            '+CelestialTater': {
+                species: 'Armaldo', ability: 'Unburden', item: 'White Herb', gender: 'M',
+                moves: ['Drain Punch', 'Stone Edge', 'MegaHorn'],
```
Comment (HoeenCoder) [inline mods/sgssb/scripts.js]: Moves are case-sensitive. Use `Megahorn` not `MegaHorn`

**Review by HoeenCoder — CHANGES_REQUESTED** [review]
Overall it looks and works great, just fix these few issues and your good to go.

**Review by HoeenCoder — APPROVED** [review]
Looks good thanks KM

**Comment by HoeenCoder** [comment]
We have to resolve conflicts before merging though.

**Comment by HoeenCoder** [comment]
Well apparently git history doesn't like me. My suggestions is to branch off of a up-to-date copy of master, make edits, and then open an new PR as I can't get this to work.

**Pull request closed without merging**